client/app/components/ParameterValueInput.less

```diff
@@ -1,50 +1,80 @@
 @import '~antd/lib/input-number/style/index'; // for ant @vars
 
+@input-dirty: #fffce1;
+
 .parameter-input {
   display: inline-block;
   position: relative;
-  width: 195px;
 
-  &[data-dirty] {
-    .@{ant-prefix}-input {
-      padding-right: 58px !important;
-    }
+  .@{ant-prefix}-input[type="text"] {
+    width: 195px;
   }
 
-  .parameter-apply-button {
-    position: absolute !important;
-    height: 25px !important;
-    width: 50px;
-    top: 5px;
-    right: 5px;
+  &[data-dirty] {
+    .@{ant-prefix}-input, // covers also ant date component
+    .@{ant-prefix}-input-number,
+    .@{ant-prefix}-select-selection {
+      background-color: @input-dirty;
+    }
   }
 }
 
-.parameter-input-number {
-  display: inline-block;
+.parameter-container {
```

Comment:
This CSS seems more coupled to the `Parameters` component than to `ParameterValueInput`, guess it should be moved when we migrate the first, right?